setup/verify_storage.py

```
@@ -16,94 +16,110 @@
     print("Database Storage Verification")
     print("=" * 60)
 
     # Get counts
-    metadata_count = db.collection('arxiv_metadata').count()
-    chunks_count = db.collection('arxiv_abstract_chunks').count()
-    embeddings_count = db.collection('arxiv_abstract_embeddings').count()
+    papers_count = db.execute_query("RETURN LENGTH(arxiv_metadata)")[0]
+    embeddings_count = db.execute_query("RETURN LENGTH(arxiv_abstract_embeddings)")[0]
+    structures_count = db.execute_query("RETURN LENGTH(arxiv_structures)")[0]
 
     print(f"\nCollection Counts:")
```

Ruff (0.13.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)